source/includes/api/_areas_v2.md

```diff
@@ -257,17 +324,17 @@ The following parameters are provided to the email service and can be used in th
 * `id` : the ID of the area.
 * `name` : the name of the area.
 * `location` : an alias for the name of the area (contains the same as the `name` parameter).
-* `subscriptions_url` : the URL to manage the areas in the frontend (example: [https://staging.globalforestwatch.org/my-gfw](https://staging.globalforestwatch.org/my-gfw)).
-* `dashboard_link` : the link to the area dashboard (example: [https://staging.globalforestwatch.org/dashboards/aoi/5d517b3fb8cfd4001061d0b2](https://staging.globalforestwatch.org/dashboards/aoi/5d517b3fb8cfd4001061d0b2)).
-* `map_link` : the "view on map" for the area (example: [https://staging.globalforestwatch.org/map/aoi/5d517b3fb8cfd4001061d0b2](https://staging.globalforestwatch.org/map/aoi/5d517b3fb8cfd4001061d0b2)).
+* `subscriptions_url` : the URL for managing areas of interest in the flagship application (example: [https://staging.globalforestwatch.org/my-gfw](https://staging.globalforestwatch.org/my-gfw)).
+* `dashboard_link` : the URL for the area dashboard (example: [https://staging.globalforestwatch.org/dashboards/aoi/5d517b3fb8cfd4001061d0b2](https://staging.globalforestwatch.org/dashboards/aoi/5d517b3fb8cfd4001061d0b2)).
+* `map_link` : the "view on map" URL for this area (example: [https://staging.globalforestwatch.org/map/aoi/5d517b3fb8cfd4001061d0b2](https://staging.globalforestwatch.org/map/aoi/5d517b3fb8cfd4001061d0b2)).
 * `image_url` : the URL for the image associated with the area.
 * `tags` : a string containing the AOI tags, comma-separated.
 
 (`5d517b3fb8cfd4001061d0b2` is an example of an area ID).
 
 ### Implementation details
 
-POST of a new area always starts by creating the area and, taking into account the area attributes, it might also create a subscription which will then be associated with the area. The area's `subscriptionId` attribute will then be updated with the id of the created subscription if that's the case. The created area is then returned.
+POST of a new area always starts by creating the area. Then, taking into account the area attributes, it might also create a subscription which will then be associated with the area. The area's `subscriptionId` attribute will then be updated with the id of the created subscription if that's the case. The created area is then returned.
```

Comment:
Before, it said that area v2 was intended to replace area v1 + subscriptions. But it sounds like subscriptions are still created in area v2? This could use clarification.

Reply:
Yes - I think the idea I missed here is that Areas still make use of Subscriptions for actually sending notifications to users. Once you are aware of this, it's easier to understand that yes, subscriptions are still created in areas v2.

I've added a section to the introduction called "Interaction between Areas and Subscriptions" which should clarify this interaction.